packages/explat-client-react-helpers/package.json

```diff
@@ -26,15 +26,14 @@
 	},
 	"dependencies": {
 		"@automattic/explat-client": "workspace:^",
-		"react": "^17.0.2",
-		"tslib": "^2.3.0"
+		"react": ">=16.8 <18",
```

Comment:
@testing-library/react-hooks doesn't yet support React 18:
https://github.com/testing-library/react-hooks-testing-library/issues/654

Reply:
WCCom seems to use 16.14 so we should be good here.

Reply:
Also, is there a reason why React 18 should be excluded? The Explat bindings should work with v18 equally well as with an earlier v16 or v17, shouldn't they?

Reply:
Unfortunately @testing-library/react-hooks doesn't yet support React 18:
https://github.com/testing-library/react-hooks-testing-library/issues/654

My unit tests are getting errors where I try to use it because the testing-library is incompatible with React 18.